docs/02-get-started/02-deploy-expose-function.md

```diff
@@ -0,0 +1,158 @@
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+kubectl get functions hello-world 
+```
+
+The operation was successful if the statuses for **CONFIGURED**, **BUILT**, and **RUNNING** are `True`.
+
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+The operation was successful if the Function **Status** changed from `DEPLOYING` to `RUNNING`.
+
+> **NOTE:** You might need to wait a few seconds for the status to change.
+  </details>
+</div>
+
+## Expose the Function
+
+After we've got our `hello-world` Function deployed, we might want to expose it outside our cluster so that it's available for other external services.
+
+> **CAUTION:** Exposing a Function to the outside world is always a potential security vulnerability, so tread carefully.
+
+First, let's create an [APIRule](../05-technical-reference/00-custom-resources/apix-01-apirule.md) for the Function.
+
+<div tabs name="Expose the Function" group="deploy-expose-function">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+cat <<EOF | kubectl apply -f -
+  apiVersion: gateway.kyma-project.io/v1alpha1
+  kind: APIRule
+  metadata:
+    name: hello-world
+    namespace: default
+  spec:
+    gateway: kyma-gateway.kyma-system.svc.cluster.local
+    rules:
+      - accessStrategies:
+        - config: {}
+          handler: allow
+        methods:
+          - GET
+          - POST
+          - PUT
+          - PATCH
+          - DELETE
+          - HEAD
+        path: /.*
+    service:
+      host: hello-world.$CLUSTER_DOMAIN
+      name: hello-world
+      port: 80
+EOF
+```
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+1. In your Function's view, go to the **Configuration** tab.
+2. Click on **Expose Function +**.
+3. Provide the **Name** (`hello-world`) and **Subdomain** (`hello-world`) and click **Create**.
+
+> **NOTE:** Alternatively, from the left navigation go to **APIRules**, click on **Create apirules +**, and continue with step 3, selecting the appropriate **Service** from the dropdown menu.
+  </details>
+</div>
+
+### Verify the Function exposure
+
+Now let's verify that the Function has been exposed successfully.
+
+<div tabs name="Access the Function" group="deploy-expose-function">
+  <details open>
+  <summary label="kubectl">
+  kubectl
+  </summary>
+
+Run:
+
+```bash
+curl https://hello-world.$CLUSTER_DOMAIN
+```
+
+The operation was successful if the call returns `Hello Serverless`.
+
+  </details>
+  <details>
+  <summary label="Kyma Dashboard">
+  Kyma Dashboard
+  </summary>
+
+In your Function's **Configuration** tab, click on the APIRule's **Hostname**.
+This will open the Function's external address as a new page.
+
+> **NOTE:** Alternatively, from the left navigation go to **APIRules**, and click on the **Host** URL there.
+
+The operation was successful if the page says `Hello World!`.
```

Comment:
I'm not super happy about it either, but via kubectl, the source code for the function is prepared "behind the scenes" and I don't even see it, and via Dashboard, the default sample code is also already there, and yes, they are different... In the dashboard it's "Hello World" (a standard), and via kubectl it's "Hello Serverless".
To unify these two I'd have to modify the code in the Dashboard, but it's not the purpose of the guide and it would be just an extra unnecessary step, so I decided to leave it as is.

Reply:
Fully agreed. Weird though ^^

Reply:
We could talk to Wookiees to change it (or I can look for it myself when I'm back, if I'm reminded :D ), as I think Wookiees have more important things to do right now :D